### PR TITLE
Small improvements

### PR DESCRIPTION
**Pull request by MatusKysel**

### CODE REVIEW

**Review by kstdl — APPROVED** [review]

File: libraries/cli/include/cli/config_jsons/testnet/testnet_config.json [hunk 1]
```diff
@@ -71,14 +71,6 @@
...
             "name": "SUMMARY",
             "verbosity": "INFO"
-          },
-          {
-            "name": "PBFT_MGR",
-            "verbosity": "INFO"
-          },
-          {
-            "name": "DAG_PROPOSER",
-            "verbosity": "INFO"
```
Comment (JakubFornadel) [inline libraries/cli/include/cli/config_jsons/testnet/testnet_config.json:81]: why did you remove those 2 channels ?

**Review by JakubFornadel — CHANGES_REQUESTED** [review]
.

File: libraries/cli/include/cli/config_jsons/testnet/testnet_config.json (hunk 1 above)
Comment (JakubFornadel) [reply libraries/cli/include/cli/config_jsons/testnet/testnet_config.json:81]: If there are some spamming messages, create issue with those so we can fix it, because it is supposed to log only info when user creates vote, dag block or pbft block which are all rewarded, not that frequent and definitely should not be spamming. This is not right solution

Comment (JakubFornadel) [reply libraries/cli/include/cli/config_jsons/testnet/testnet_config.json:81]: so what are useful logs for community users in your opinion then ? If not such most basic thing as logging when he created pbft or dag block then I dont know what ? If user monitors his logs, he probably wants to see at least some basic logs about his activity + errors

Comment (JakubFornadel) [reply libraries/cli/include/cli/config_jsons/testnet/testnet_config.json:81]: yeah I think your approach is wrong too :) If it was some problem that takes long to solve then yes, this is probably 1 or 2 logs where we simply set verbosity to debug instead of info but you rather turn it off completely ?

Comment (JakubFornadel) [reply libraries/cli/include/cli/config_jsons/testnet/testnet_config.json:81]: I will do it no problem just give me the messages as you already investigated it ?

Comment (JakubFornadel) [reply libraries/cli/include/cli/config_jsons/testnet/testnet_config.json:81]: > and if you want new channels enable for community user please join discord and explain then why you enabled it

do you know that you did approve it ? https://github.com/Taraxa-project/taraxa-node/pull/2196. Did you read the description ? So you enabled it too in a fact :)

**Review by JakubFornadel — APPROVED** [review]
